src/location/ob_part_desc.rs

```diff
@@ -723,7 +723,7 @@ impl ObKeyPartDesc {
 
     // TODO: check if murmur2 hash value is correct(java sdk)
     pub fn long_hash(value: i64, hash_code: i64) -> Result<i64> {
```

Comment:
add long_hash test

Reply:
long_hash is so simple that test is not needed here...